modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -258,6 +272,14 @@ public function buildForm(array $form, FormStateInterface $form_state, $step_id
     }
     $form['actions'] = $this->actions($form, $form_state);
 
+    // Ensure checkout flows have the correct cacheable metadata so they can
+    // be cached properly.
+    $config_parent = $this->entityTypeManager->getStorage('commerce_checkout_flow')->load($this->entityId);
```

Comment:
Let's go with $parent_entity, feels a bit clearer, $config_parent makes me think it's a CMI object.